src/lib/asn1/ASN1Reader.cpp

```diff
@@ -113,6 +113,7 @@ CHIP_ERROR ASN1Reader::EnterContainer(uint32_t offset)
     mElemStart = Value + offset;
     if (!IndefiniteLen)
     {
+        VerifyOrReturnError(mBufEnd - Value >= ValueLen, ASN1_ERROR_VALUE_OVERFLOW);
```

Comment:
Instance data such as `Value` and `ValueLen` should follow the coding conventions and be named `mValue` and `mValueLen` respectively.

Reply:
Will do that in a separate PR.